### PR TITLE
Feature/nunit3 add possibility to use fullname attribute as test name

### PR DESCRIPTION
**Pull request by VladislavKostyukevich**
# PR Details

Added possibility To choose test name node before importing NunitV3 file (Feature: Test Name will be used by default)

## Related Issue

#27 

## How Has This Been Tested

1. Import Nunit v3 file with Feature: Test Name option
1. See the imported test names
1. Import Nunit v3 file with Test Class Name option
1. See the imported test names

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/main/java/main/model/db/imports/BaseImporter.java [hunk 1]
```diff
@@ -80,7 +80,10 @@ void addLogToImport(String log) throws AqualityException {
 
     private void createTestRun() throws AqualityException {
         if(testRun.getId() != null){
-            this.testRun = controllerFactory.getHandler(testRun).get(testRun, false, 1).get(0);
+            TestRunDto template = new TestRunDto();
+            template.setProject_id(testRun.getProject_id());
+            template.setId(testRun.getId());
+            testRun = controllerFactory.getHandler(template).get(template, false, 1).get(0);
```
Comment (knysh) [inline src/main/java/main/model/db/imports/BaseImporter.java]: Is it necessary to check that size is more than 1?

File: src/main/java/main/model/db/imports/SAXHandlers/NUnitV3Handler.java [hunk 2]
```diff
@@ -166,6 +169,16 @@ public void characters(char[] ch, int start, int length) {
         }
     }
 
+    private void setTestName(Attributes attributes) throws SAXException {
+        if(testNameNodeType == TestNameNodeType.featureNameTestName) {
```
Comment (knysh) [inline src/main/java/main/model/db/imports/SAXHandlers/NUnitV3Handler.java]: if would change it to switch\case

File: src/main/java/main/controllers/Project/SuiteController.java [hunk 3]
```diff
@@ -46,6 +46,17 @@ public TestSuiteDto create(TestSuiteDto template) throws AqualityException {
         return get(template, false);
     }
 
+    public TestSuiteDto get(String name, Integer projectId) throws AqualityException {
+        TestSuiteDto template = new TestSuiteDto();
+        template.setName(name);
+        template.setProject_id(projectId);
+        try{
+            return get(template, false).get(0);
+        } catch (Exception e) {
```
[the remark this answers is not in the document]
Comment (VladislavKostyukevich) [reply src/main/java/main/controllers/Project/SuiteController.java]: fixed

File: src/main/java/main/model/db/imports/BaseImporter.java (hunk 1 above)
Comment (VladislavKostyukevich) [reply src/main/java/main/model/db/imports/BaseImporter.java]: fixed

File: src/main/java/main/model/db/imports/SAXHandlers/NUnitV3Handler.java (hunk 2 above)
Comment (VladislavKostyukevich) [reply src/main/java/main/model/db/imports/SAXHandlers/NUnitV3Handler.java]: fixed

**Pull request merged**